From c6794260bbea14a7fac703e52228d5d15c9a22d0 Mon Sep 17 00:00:00 2001
From: shindesaahil <sahil.shinde@trunexa.com>
Date: Sun, 18 Aug 2024 11:06:35 +0530
Subject: [PATCH 1/5] iMX8M: soc: allow dtb override

---
 iMX8M/soc.mak | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/iMX8M/soc.mak b/iMX8M/soc.mak
index c505dea..a45a015 100644
--- a/iMX8M/soc.mak
+++ b/iMX8M/soc.mak
@@ -144,10 +144,7 @@ u-boot-atf-tee.bin: u-boot.bin bl31.bin $(TEE)
 clean:
 	@rm -f $(MKIMG) u-boot-atf.bin u-boot-atf-tee.bin u-boot-spl-ddr.bin u-boot.itb u-boot.its u-boot-ddr3l.itb u-boot-ddr3l.its u-boot-spl-ddr3l.bin u-boot-ddr4.itb u-boot-ddr4.its u-boot-spl-ddr4.bin u-boot-ddr4-evk.itb u-boot-ivt.itb u-boot-ddr4-evk.its $(OUTIMG)
 
-dtb = evk.dtb
-$(dtb):
-	./$(DTB_PREPROC) $(PLAT)-evk.dtb $(dtb) $(dtbs)
-
+dtb = $(dtbs)
 u-boot.itb: $(dtb)
 	./$(PAD_IMAGE) $(TEE)
 	./$(PAD_IMAGE) bl31.bin
-- 
2.34.1

